src/shared/ui/Button/Button.tsx

```diff
@@ -1,13 +1,14 @@
+import { FC } from 'react';
```

Comment:
Let's start using storybook and Button it's a good candidate to be added 
Let's create a Button.stroies.tsx and put some logic for rendering Button with different properties, so you can some other props along with onClick and children
Use this as an example - https://github.com/Aleksey-Timoschenko/production-app/blob/main/src/shared/ui/Button/Button.stories.tsx